src/routes/applications.ts

```diff
@@ -334,6 +339,16 @@ const createApplicationsRouter = (
     }),
   );
 
+  router.get(
+    '/collaborators/applications',
```

Comment:
not sure about the best way to name this endpoint?

Reply:
I think it's fine for now

Reply:
oops yes I forgot. will add now 👍

Reply:
@blabadi added route to swagger